pontoon/sync/tests/test_core.py

```diff
@@ -340,6 +340,26 @@ def test_multiple_authors(self):
             os.path.join(FAKE_CHECKOUT_PATH, self.translated_locale.code)
         )
 
+    def test_author_with_multiple_contributions(self):
+        """
+        Tests if author with multiple contributions occurs once in commit message.
+        """
+        author = UserFactory.create()
+        self.changeset.commit_authors_per_locale = {
+            self.translated_locale.code: [author, author]
+        }
+        self.db_project.repository_for_path = Mock(return_value=self.repository)
+
+        commit_changes(self.db_project, self.vcs_project, self.changeset)
+
+        self.repository.commit.assert_called_with(
+            CONTAINS(author.display_name),
+            author,
+            os.path.join(FAKE_CHECKOUT_PATH, self.translated_locale.code)
+        )
+        commit_message = self.repository.commit.mock_calls[0][1][0]
+        assert_equal(len(set(re.finditer(author.display_name, commit_message))), 1)
```

Comment:
If you wrap it in a set it'll remove the duplicates even if there were any in the message. Also, `re` is overkill for this, you should just be able to do `commit_message.count(author.display_name)`.